app/assets/javascripts/discourse/views/topic-footer-main-buttons.js.es6

```diff
@@ -7,6 +7,7 @@ export default ContainerView.extend({
   @on('init')
   createButtons() {
     const mobileView = this.site.mobileView;
+    const { siteSettings } = this;
```

Comment:
Not sure if this variable is needed right now since we're only fetching from siteSettings in a single place.